biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java

```diff
@@ -57,11 +59,11 @@
  */
 @BndPlugin(name = "BndPomRepository")
 public class BndPomRepository extends BaseRepository
-	implements Plugin, RegistryPlugin, RepositoryPlugin, Refreshable, Actionable, Closeable {
+	implements Plugin, RegistryPlugin, RepositoryPlugin, Refreshable, Actionable, Closeable, Prepare {
 	private static final String	MAVEN_REPO_LOCAL	= System.getProperty("maven.repo.local", "~/.m2/repository");
 	private static final int	DEFAULT_POLL_TIME	= 300;
 
-	private boolean				inited;
+	private Boolean				inited				= false;
```

Comment:
Why?

Reply:
I used the init to sync on  at one point and just forgot to switch it back.